### DISCUSSION TITLE
Neovim File Icons

### DISCUSSION THREAD

**Discussion by 0xJWLabs** [catppuccin/catppuccin#2384] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

I'm suggesting to create like own font ligature to create port of vscode-icons like nvim web devicons

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by noahbald** [comment]
https://github.com/catppuccin/catppuccin/discussions/2238
I tried making a start on this here without too much luck.
I was trying to get stuff automated with vscode-icons but I'm sure if you did it manually you could get somewhere.

**Reply by 0xJWLabs** [reply]
Uhm, I can't use font-patcher

**Comment by uncenter** [comment]
https://github.com/rachartier/tiny-devicons-auto-colors.nvim might solve this? The example setup instructions are also already for Catppuccin!

**Reply by MuntasirSZN** [reply]
Its just for colors. We want the icons too!

**Reply by sgoudham** [reply]
Implementing the icons for Neovim requires a lot more effort (font patching) than what currently exists for JetBrains and VSCode so I'm not surprised that many people haven't tried to tackle it.

**Comment by sgoudham** [comment]
I didn't realise #2238 had been raised already, going to close this as a duplicate for the time being.
